cupy/core/fusion.py

```diff
@@ -753,15 +752,15 @@ def __repr__(self):
         return '<Fusion \'{}\'>'.format(self.name)
 
     def __call__(self, *args, **kwargs):
-        if _thread_local.history is None:
+        if hasattr(_thread_local, 'history'):
```

Comment:
This condition should be `not hasattr(_thread_local, 'history')`.

Reply:
Thanks, fixed.